Antidote/ContactCellView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ContactCellView.swift
+//  Antidote
+//
+//  Created by Yoba on 04/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+class ContactCellView: NSTableCellView {
+    var userAvatar: NSImageView!
+    var usernameTextField: NSTextField!
+    var lastMessageTextField: NSTextField!
+    var callButton: NSButton!
+    var optionButton: NSButton!
+    
+    override init(frame: NSRect) {
+        super.init(frame: frame)
+        
+        self.createSubviews()
+        //self.setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override func viewDidMoveToSuperview() {
+        setupConstraints()
+    }
+}
+
+fileprivate extension ContactCellView {
+    func createSubviews() {
+        userAvatar = NSImageView()
+        addSubview(userAvatar)
+        
+        usernameTextField = LabelView()
+        usernameTextField.font = NSFont(name: "System Font", size: 20)
```

Comment:
`NSFont` has a class method for that `usernameTextField.font = NSFont.systemFont(ofSize: 20)`